src/adapters/indexExchange.js

```diff
@@ -361,7 +361,7 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     }
 
     //bidmanager.setExpectedBidsCount(ADAPTER_CODE, expectedBids);
-    adloader.loadScript(cygnus_index_start());
+    adloader.loadScript(cygnus_index_start(), null, true);
```

Comment:
Not clear on how this one works, but since they don't specify a callback it might break on refreshAds. Need to verify